src/OpenTelemetry/OpenTelemetry.Customization.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net5.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="StackExchange.Redis" Version="2.2.4" />
+		<PackageReference Include="OpenTelemetry" Version="1.0.1" />
+		<PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.0.1" />
+		<PackageReference Include="OpenTelemetry.Exporter.Jaeger" Version="1.0.1" />
+		<PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.0.1" />
+		<PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="1.0.1" />
+		<PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc2" />
```

Comment:
It'd be best to not use Extensions.hosting package today, and stick with manual config. This package is expected to undergo more changes.

Reply:
ok, sure. 
It's being used for the `AddOpenTelemetryTracing()` method.  Are we supposed to use any other package? Because I see the same package is being used in this [example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/AspNetCore/Examples.AspNetCore.csproj) as well.